plat/imx/imx8m/imx8mp/imx8mp_bl31_setup.c

```diff
@@ -207,6 +207,10 @@ void bl31_platform_setup(void)
 	plat_gic_init();
 
 	imx_gpc_init();
+
+	/* Enable and reset M7 */
+	mmio_setbits_32(IMX_SRC_BASE + 0xc, SRC_SCR_M4_ENABLE_MASK);
+	mmio_clrbits_32(IMX_SRC_BASE + 0xc, SRC_SCR_M4C_NON_SCLR_RST_MASK);
```

Comment:
Does `PLAT_imx8mm` require the same?

Reply:
it has similar here
https://github.com/Qingwu-Li/trusted-firmware-a/blob/3656720985b2bfbfdf379949d4802fd645f6418d/plat/imx/imx8m/gpc_common.c#L262